jetty-ee10/jetty-ee10-webapp/src/main/java/org/eclipse/jetty/ee10/webapp/MetaData.java

```diff
@@ -485,7 +486,8 @@ public void resolve(WebAppContext context)
         resources.add(null); //always apply annotations with no resource first
         resources.addAll(_orderedContainerResources); //next all annotations from container path
         resources.addAll(_webInfClasses); //next everything from web-inf classes
-        resources.addAll(getWebInfResources(isOrdered())); //finally annotations (in order) from webinf path 
+        resources.addAll(getWebInfResources(isOrdered())); //finally annotations (in order) from webinf path
+        resources = resources.stream().distinct().collect(Collectors.toList()); // filter out possible duplicates
```

Comment:
@janbartel @jackeri  I'm not opposed to defensively removing duplicates, but I think we should warn.  Somethine like:
```suggestion
        List<Resource> nodups = resources.stream().distinct().collect(Collectors.toList()); // filter out possible duplicates
        if (nodups.length() < resources.length())
        {
            LOG.warning("Duplicate jars: " + resources);
            resources = nodups;
        } 
```

Reply:
I'm not keen on this, as it is an error to have duplicates on the classpath, which is where this derives from.